File: Basics_code_igenuine/php/myweb/works/ams.php
 <?php
function armstrongNo($num1)
{
	$sum=0;
	$num2=$num1;
		$len=count(str_split($num1));
		while($num2!=0)
		{
			$d=$num2%10;
			$sum+=(int)pow($d,$len);
			$num2=$num2/10;
		}
		if($sum==$num1)
		{
			echo $num1." is Amstrong";
		}
		else
		{
			echo $num1." is not Amstrong";
		}
}
?>
<html>
<head></head>
<body>
<?php
if(!isset($_POST['submit']))
{
?>
<form method="post" action="ams.php">
Enter the number :<br>
<input type="text" name="num"size="4"/><br>
<input type="submit" name="submit"value="Submit"/><br>
</form>
<?php
}
	else
	{
	$num=(int)$_POST['num'];
	armstrongNo($num); 
	}
?>
</body>
</html>

File: Basics_code_igenuine/java/works/ams.java
import java.util.Scanner;
import java.lang.Math;
public class ams
{
	public static void main(String args[])
	{
		int num1,num2,num3,d,sum=0;
		int len=0;
		Scanner sc=new Scanner(System.in);
		System.out.println("Enter the number:");
		num1=sc.nextInt();
		sc.close();
		num2=num1;
		num3=num1;
		while(num3!=0)
		{
			num3=num3/10;
			len++;
		}
		while(num2!=0)
		{
			d=num2%10;
			sum+=Math.pow(d,len);
			num2=num2/10;
		}
		if(sum==num1)
		{
			System.out.println(num1+" is Amstrong");
		}
		else
		{
			System.out.println(num1+" is not Amstrong");
		}
	}
}